source/preprocessor.tex

```diff
@@ -1402,6 +1403,7 @@
 \grammarterm{pp-tokens} in a pragma directive. The original four preprocessing
 tokens in the unary operator expression are removed.
 
+\pnum
 \begin{example}
 
```

Comment:
Delete this pointless newline while you're at it. We got over whatever made us have those back in the day.